docs/reference/sql/tql.md

```diff
@@ -24,10 +24,93 @@ Return the per-second rate for all time series with the `http_requests_total` me
 TQL eval (1677057993, 1677058993, '1m') rate(prometheus_http_requests_total{job="prometheus"}[5m]);
 ```
 
+Will get result just like other normal SQL queries.
+
+## EXPLAIN
+
+`EXPLAIN` shows the logical plan and execution plan of one PromQL query. The syntax is:
+
+```
+TQL EXPLAIN expr;
+```
+
+For example, to explain the PromQL `sum by (instance) (rate(node_disk_written_bytes_total[2m])) > 50`, we can use
+
+```
+TQL EXPLAIN sum by (instance) (rate(node_disk_written_bytes_total[2m])) > 50;
+```
+
+Notice that since the given query won't be actual executed, thus the triple `(start, end, step)` is not required. But you can still provide one like in `TQL EVAL`:
```

Comment:
```suggestion
Notice that since the given query won't be actually executed, the triple `(start, end, step)` is not necessary. But you can still provide it like in `TQL EVAL`:
```